app/portainer/components/forms/kubernetes-app-git-form/kubernetes-app-git-form.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+const kubernetesAppGitForm = {
+  templateUrl: './kubernetes-app-git-form.html',
+  bindings: {
+    onChange: '<',
+    onChangeRef: '<',
```

Comment:
instead of passing `onChangeRef` from parent, define it on the controller and call `onChange` in it. that's what is happening in the parent anyway